### ISSUE TITLE
Standardize responses

### ISSUE THREAD

**Issue by mgxd** [mgxd/migas-py#12]
3 different scenarios with 3 different response bodies.

```
>>> add_project(project='nipreps/fmriprep', project_version='21.0.2')

{'data': {'add_project': {'bad_versions': [],
    'cached': False,
    'latest_version': '21.0.2',
    'message': '',
    'success': True}}}

>>> add_project(project='nipreps/fmriprep', project_version='21.0.2')  # server unavailable

 {'success': False,
  'errors': [{'message': 'Connection to server timed out.'}]}

>>> add_project(project='nipreps/fmriprep', project_version='21.0.2')  # requests overload

{'data': None, 'errors': [{'message': 'Too many requests, wait a minute.'}]}
```

These should all converge to something similar, to allow easy filtering

**Comment by mgxd** [comment]
After #13 - the `data` field should always be present; it now can be parsed in a separate PR
